### PR TITLE
Fix to address distinct bug when sorting inbox messages on list page.

### PR DESCRIPTION
**Pull request by emullaney**

### CODE REVIEW

**Comment by emullaney** [comment]
This is ready for review @rebecca-caktus. Thanks for the assistance working this out.

File: tracpro/msgs/views.py [hunk 1]
```diff
@@ -112,7 +112,21 @@ class List(OrgPermsMixin, SmartListView):
...
-            qs = qs.order_by('contact', '-created_on').distinct('contact')
+
+            qs = qs.order_by('contact', '-created_on')
+            qs = qs.distinct('contact')
+
+            return qs
+
+        def order_queryset(self, qs):
+            """
+            Override the order_queryset() method from smartmin
```
Comment (rebecca-caktus) [inline tracpro/msgs/views.py]: I don't think this comment is very helpful - maybe something more specific about how the default ordering mechanisms won't work due to the distinct query on contact?

Should we also consider applying validation of the `_order` parameter?

Comment (emullaney) [inline tracpro/msgs/views.py]: Fixed!

**Comment by rebecca-caktus** [comment]
:rose:

**Pull request merged**